site/docs/tools/cli.md

```diff
@@ -134,19 +149,33 @@ Usage: nessie contents [OPTIONS] [KEY]...
   namespace what is included.
 
 Options:
-  -l, --list            list tables
-  -d, --delete          delete a table
-  -s, --set             modify a table
-  -c, --condition TEXT  Conditional Hash. Only perform the action if branch
-                        currently points to condition.
-
-  -r, --ref TEXT        branch to list from. If not supplied the default
-                        branch from config is used
-
-  -m, --message TEXT    commit message
-  --help                Show this message and exit.
-!!! caution
-    Please note that we're actively redefining the cli to better match Git syntax. You should expect that this syntax will change shortly.
+  -l, --list                      list tables
+  -d, --delete                    delete a table
+  -s, --set                       modify a table
+  -c, --condition TEXT            Conditional Hash. Only perform the action if
+                                  the branch currently points to the hash
+                                  specified by this option.
+  -r, --ref TEXT                  branch to list from. If not supplied the
+                                  default branch from config is used
+  -m, --message TEXT              commit message
+  -t, --type TEXT                 entity types to filter on, if no entity
+                                  types are passed then all types are returned
+  --query, --query-expression TEXT
+                                  Allows advanced filtering using the Common
```

Comment:
The CEL references are good - I like it. It's just hard to read here with the small column size. Can we widen the column at least and add some newlines to the message?